code/game/mob/living/carbon/human/human_defense.dm

```diff
@@ -105,7 +103,6 @@ bullet_act
 				if (!(H.real_name in map.warrants) && prob(probtoissue))
 					map.warrants += H.real_name
 					H.gun_permit = 0
-					H.bail_price += 500
```

Comment:
This doesn't seem related

Reply:
this is not related no clue why this changed